src/pubd/manager.rs

```diff
@@ -459,15 +459,15 @@ mod tests {
         let session = session_dir(&d);
 
         // Should not include
-        assert!(!session_dir_contains_serial(&session, 0));
+        assert!(!session_dir_contains_serial(&session, RRDP_FIRST_SERIAL));
```

Comment:
Would it be worth adding a check that there is no zero serial present?

Reply:
it would - done